### DISCUSSION TITLE
Unmute notification should not restore to 100% volume

### DISCUSSION THREAD

**Discussion by smoogipoo** [ppy/osu#16190] [Q&A]
I have my in-game volume set to 30% and system set to 50%. The unmute notification restoring volume to 100% is bad enough that I can't use it for fear of going deaf.

Other users share the same feeling: https://github.com/ppy/osu/pull/15999#issuecomment-998478459

Restoring to 20% or so seems like a better default.

**Comment by peppy** [comment]
May be good to do a survey and figure what people have their volume set to. Sounds like we may want to adjust the defaults and potentially the range.
